### PR TITLE
Meta: Workflow improvements for RFCs

### LINKED ISSUE #7: Add advice to include a Rendered link at the top of each RFC
See https://github.com/bevyengine/rfcs/pull/1 for an example.

You need to point to the file in the branch of the originating fork. Use the label, not a specific commit, so it stays fresh.

This is useful for readers who prefer reading formatted markdown.

### LINKED ISSUE #8: Accepted and implemented categorization
It would be nice to be able to tell at a glance which designs still have work to do.

Folders is a decent approach to this, but we could also just use a file prefix.

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Closes #8.
Closes #7.

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
Also closes #7.

File: README.md [hunk 1]
```diff
@@ -27,9 +27,11 @@ They don't need to be perfect, complete, or even very good when you submit them.
 1. Fork this repository and create a new branch for your new RFC.
 2. Copy `template.md` into the `rfcs` folder and rename it to `my_feature.md`, where `my_feature` is a unique identifier for your feature.
 3. Fill out the RFC template with your vision for `my_feature`.
-4. [Create a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) in this repo, with a one-sentence description of what the RFC is about.
+4. [Create a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) in this repo. The first comment should include:
+   1. A one-sentence description of what the RFC is about.
+   2. A link to the "rendered" form of the `my_feature.md` file. To do so, link directly to the file on your own branch, so then the link stays up to date as the file's contents changes. See #1 for an example of what this looks like.
```
Comment (cart) [inline README.md:32]: `#1` is ambiguous here. Maybe used bulleted items instead?

Comment (alice-i-cecile) [reply README.md:32]: Markdown automatically converts nested numbered lists into i. ii. etc notation :) See: https://github.com/bevyengine/rfcs/blob/c3175d40ced82e764dec6a5190d7e6787072d8d2/README.md

Comment (cart) [reply README.md:32]: Cool cool. Carry on!

**Pull request merged**